website/releases/release-0.15.0.md

```diff
@@ -0,0 +1,339 @@
+sidebar_position: 2
+layout: releases
+toc: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## [Release 0.15.0](https://github.com/apache/hudi/releases/tag/release-0.15.0) ([docs](/docs/quick-start-guide))
+
+Apache Hudi 0.15.0 release brings enhanced engine integration, new features, and improvements in several areas. These
+include Spark 3.5 and Scala 2.13 support, Flink 1.18 support, better Trino Hudi native connector support with newly
+introduced Hadoop-agnostic storage and I/O abstractions. We encourage users to review
+the [Release Highlights](#release-highlights) and [Migration Guide](#migration-guide-overview) down below on
+relevant [module and API changes](#module-and-api-changes),  [breaking changes](#breaking-changes) and
+[behavior changes](#behavior-changes) before using the 0.15.0 release.
+
+## Migration Guide
+
+This release keeps the same table version (`6`) as [0.14.0 release](/releases/release-0.14.0), and there is no need for
+a table version upgrade if you are upgrading from 0.14.0. There are a
+few [module and API changes](#module-and-api-changes),
+[breaking changes](#breaking-changes), and [behavior changes](#behavior-changes) as
+described below, and users are expected to take action accordingly before using 0.15.0 release.
+
+:::caution
+If migrating from an older release (pre-0.14.0), please also check the upgrade instructions from each older release in
+sequence.
+:::
+
+### Bundle Updates
+
+#### New Spark Bundles
+
+We have expanded Hudi support to Spark 3.5 with two new bundles:
+
+- Spark 3.5 and Scala
+  2.12: [hudi-spark3.5-bundle_2.12](https://mvnrepository.com/artifact/org.apache.hudi/hudi-spark3.5-bundle_2.12)
+- Spark 3.5 and Scala
+  2.13: [hudi-spark3.5-bundle_2.13](https://mvnrepository.com/artifact/org.apache.hudi/hudi-spark3.5-bundle_2.13)
+
+#### New Utilities Bundles for Scala 2.13
+
+Besides adding a bundle for Spark 3.5 and Scala 2.13, we have added new utilities bundles to use with Scala
+2.13, [hudi-utilities-bundle_2.13](https://mvnrepository.com/artifact/org.apache.hudi/hudi-utilities-bundle_2.13)
+and [hudi-utilities-slim-bundle_2.13](https://mvnrepository.com/artifact/org.apache.hudi/hudi-utilities-slim-bundle_2.13).
+
+#### New and Deprecated Flink Bundles
+
+We have expanded Hudi support to Flink 1.18 with a new
+bundle, [hudi-flink1.18-bundle](https://mvnrepository.com/artifact/org.apache.hudi/hudi-flink1.18-bundle). This release
+removes Hudi support on Flink 1.13.
+
+### Module and API Changes
+
+#### Hudi Storage and I/O Abstractions
+
+This release introduces new storage and I/O abstractions that are Hadoop-agnostic to improve integration with query
+engines, including [Trino](https://trino.io/), which uses its own native File System APIs. Core Hudi classes
+including [`HoodieTableMetaClient`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java),
```

Comment:
should we link to permalinks?
for eg:
https://github.com/apache/hudi/blob/38832854be37cb78ad1edd87f515f01ca5ea6a8a/hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

Reply:
Addressed.  The `release-0.15.0` tag/branch should always be available; still, permalinks are better.